org.adempiere.base/src/org/compiere/model/MDocumentStatus.java

```diff
@@ -59,16 +59,33 @@ public MDocumentStatus(Properties ctx, ResultSet rs, String trxName) {
 	 *	@return array of document status
 	 */
 	public static MDocumentStatus[] getDocumentStatusIndicators(Properties ctx, int AD_User_ID, int AD_Role_ID)
+	{
+		return getDocumentStatusIndicators(ctx,AD_User_ID,AD_Role_ID);
```

Comment:
this is missing the null trxName parameter and will cause infinite loop.

Reply:
Fixed this.